tutorials/financial-tick-data/financial-tick-compress.md

```diff
@@ -8,77 +8,70 @@ layout_components: [next_prev_large]
 content_group: Analyze financial tick data
 ---
 
-# Set up compression and compress the dataset
-
-You have now seen how to create a hypertable for your financial tick
-data and query it. When ingesting a dataset like this
-is seldom necessary to update old data and over time the amount of
-data in the tables grows. Over time you end up with a lot of data and
-since this is mostly immutable you can compress it to save space and
-avoid incurring additional cost.
-
-It is possible to use disk-oriented compression like the support
-offered by ZFS and Btrfs but since TimescaleDB is build for handling
-event-oriented data (such as time-series) it comes with support for
-compressing data in hypertables.
-
-TimescaleDB compression allows you to store the data in a vastly more
-efficient format allowing up to 20x compression ratio compared to a
-normal PostgreSQL table, but this is of course highly dependent on the
-data and configuration.
-
-TimescaleDB compression is implemented natively in PostgreSQL and does
-not require special storage formats. Instead it relies on features of
-PostgreSQL to transform the data into columnar format before
-compression. The use of a columnar format allows better compression
-ratio since similar data is stored adjacently. For more details on how
-the compression format looks, you can look at the [compression
-design][compression-design] section.
-
-A beneficial side-effect of compressing data is that certain queries
-are significantly faster since less data has to be read into
-memory.
+import TutorialsHypercoreIntro from "versionContent/_partials/_tutorials-hypercore-intro.mdx"
+
+# Compress your data using $HYPERCORE
+
+<TutorialsHypercoreIntro />
+
+## Optimize your data in the $COLUMNSTORE
+
+To compress the data in the `crypto_ticks` table, do the following:
 
 <Procedure>
 
-## Compression setup
+1. Connect to your $SERVICE_LONG
+
+   In [$CONSOLE][services-portal] open an [SQL editor][in-console-editors]. The in-Console editors display the query speed.
+   You can also connect to your service using [psql][connect-using-psql].
 
-1.  Connect to the Timescale database that contains the financial tick
-    dataset using, for example `psql`.
-1.  Enable compression on the table and pick suitable segment-by and
+1.  Enable the $COLUMNSTORE on the table and pick suitable segment-by and
```

Comment:
Can we agree on one way to do this, please? Is it order-by/segment-by column or `orderby`/`segmentby` column?